src/utils/components/PodSelectorModal/selector-requirement.js

```diff
@@ -0,0 +1,96 @@
+import * as _ from 'lodash-es';
+
+import { createEquals } from '../Selector/utilts';
+
+export const requirementFromString = (string) => {
+  string = string.trim();
+
+  // "key"
+  if (/^[0-9A-Za-z/\-_.]+$/.test(string)) {
```

Comment:
Suggestion: Save these to a descriptive variable in another file to make this code more readable, then the comments won't be necessary.